ts/state/ducks/section.tsx

```diff
@@ -10,7 +10,7 @@ export enum SectionType {
   Profile,
   Message,
   Settings,
-  Moon,
+  ColorMode,
```

Comment:
not sure about the name of this

Reply:
This is what it is called in the Session Components Figma. I'm also iffy on it but we would need to discuss with Morgan and Connor

Reply:
:vomiting_face: